### PR TITLE
Add documentation regarding event condition logic

### PR DESCRIPTION
**Pull request by benfoster04**
Lines 28-30

### CODE REVIEW

File: example_config.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "guild_id": 111111111111111111,
     "info_guild_id": "Guild (server) ID. Bot will only work in this guild.",
     "ticket_users": [508391840525975553],
-    "info_ticket_users": "User IDs. The bot will create threads under channels created by those users. The currently inputed ID is for the main Tickets bot",
```
Comment (Tech-TTGames) [inline example_config.json:8]: Maybe a note on the source of the ID provided should be kept?

**Review by Tech-TTGames — CHANGES_REQUESTED** [review]

**Comment by kk5dire** [comment]
@benfoster04 I assumed it marked my last review as invalid due to the format modification, however my review does still apply. Any comments on such?

File: example_config.json (hunk 1 above)
Comment (benfoster04) [reply example_config.json:8]: If we're making this a multi-use bot maybe we should keep arrays empty and have the bot provide user feedback (via console or other means, such as through Discord) as to the lack of configuration?

Comment (benfoster04) [reply example_config.json:8]: Sorry I should probably be more clear, if this is a bot for servers other than ours, there needs to be more error checking and a broader functionality set to accomodate as such

Comment (kk5dire) [reply example_config.json:8]: > Sorry I should probably be more clear, if this is a bot for servers other than ours, there needs to be more error checking and a broader functionality set to [accommodate] as such

I feel that they should understand via the comments in the code that they are missing some config info, but some error checking could be a nice general "lets not explode the bot by erroring out" catch

**Comment by benfoster04** [comment]
> @benfoster04 I assumed it marked my last review as outdated due to the format modification, however my review does still apply. Any comments on such?

Sorry mate I've not got your review

**Comment by kk5dire** [comment]
> > @benfoster04 I assumed it marked my last review as outdated due to the format modification, however my review does still apply. Any comments on such?
> 
> Sorry mate I've not got your review

Thats alright, just making sure you noticed it as git marked mine as outdated due to the formatting update you made

File: main_utils.py [hunk 2]
```diff
@@ -25,6 +25,10 @@ async def on_channel_create(self, channel):
         """
         EXTENSION 1 + 3: Staff notes for tickets! and button stripping.
         Minor Extension 2: Safe Community Support
+
+        This event works based on bot creation logic.
+        Please ensure you add your ticket bot or approved
```
Comment (kk5dire) [inline main_utils.py:30]: #30 

> 29 + "Bot creation logic" is a bit broad, maybe say "on ticket creation events"? (Assuming you are writing about what I think you are)
30 + "Ticket creation users", and "ticket users configuration" is left unexplained, do you mean the staff teams section of the main tickets' bot, or something else? If so, specify!

**Review by kk5dire — CHANGES_REQUESTED** [review]

File: main_utils.py [hunk 3]
```diff
@@ -25,6 +25,9 @@ async def on_channel_create(self, channel):
         """
         EXTENSION 1 + 3: Staff notes for tickets! and button stripping.
         Minor Extension 2: Safe Community Support
+
+        This event works based on bot creation logic.
+        Please ensure you add your ticket bot or approved ticket creation users to the ticket users configuration.
```
[the remark this answers is not in the document]
Comment (kk5dire) [reply main_utils.py]: Resolving this as I prmoted it to a review

[the remark this answers is not in the document]
Comment (benfoster04) [reply main_utils.py]: I agree, this can be clarified. Stand by

[the remark this answers is not in the document]
Comment (benfoster04) [reply main_utils.py]: This has now been updated. Please see commit 45706e1.

**Review by Tech-TTGames — DISMISSED** [review]
LGTM

File: main_utils.py [hunk 4]
```diff
@@ -25,6 +25,10 @@ async def on_channel_create(self, channel):
         """
         EXTENSION 1 + 3: Staff notes for tickets! and button stripping.
         Minor Extension 2: Safe Community Support
+
+        This event works based on the ticket_users configuration.
+        Please ensure you add your ticket bot or approved
+         ticket creation users to the ticket users configuration.
```
Comment (kk5dire) [inline main_utils.py:31]: I still feel that "ticket creation users" and "ticket users configuration" could be better explained. 

Perhaps the former could be "ticket invoker" or "ticket creator" if relating to the creator of the ticket. 

Is the "Ticket users configuration" The panel config? If so or not, more specification would be nice

**Review by kk5dire — CHANGES_REQUESTED** [review]

File: main_utils.py (hunk 4 above)
Comment (kk5dire) [reply main_utils.py:31]: GG on the first bit although

**Comment by kk5dire** [comment]
Once my review is resolved or compromised, I'll go ahead and approve and merge

**Comment by Tech-TTGames** [comment]
Catching up to v0.0.3

File: main_utils.py (hunk 4 above)
Comment (benfoster04) [reply main_utils.py:31]: At the end of the day, while the code is confusing, the general user is not going to look into the code and understand it in the way we do. If a programmer is desperately trying to work it out beyond the notes in the configuration file, I feel like that's all it needs

**Comment by kk5dire** [comment]
Pulling updates to main branch as of #35

**Comment by benfoster04** [comment]
Closing PR in favour of new one due to major failure in brain capacity. Stand by

**Pull request closed without merging**